__mocks__/handlers/challenges.ts

```diff
@@ -0,0 +1,18 @@
+import { http, HttpResponse } from "msw";
+import { challenge, submission } from "../fixtures/challenge";
+const baseUrl = process.env.NEXT_PUBLIC_API_BASE_URL
```

Comment:
good job so far, can we have this base url defined in one place only instead of having it in every handler file